app/main.py

```diff
@@ -1,8 +1,46 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self, comfort_class: int, clean_mark: int, brand: str
```

Comment:
Better to move each parameter to a new line, like you did in CarWashStation.init to improve readability and use same style throughout the code